vendors/st/boards/stm32l475_discovery/ports/secure_sockets/iot_secure_sockets.c

```diff
@@ -148,6 +156,9 @@ typedef struct STSecureSocket
     void * pvTLSContext;                /**< The TLS Context. */
     char * pcServerCertificate;         /**< Server certificate. Set using SOCKETS_SO_TRUSTED_SERVER_CERTIFICATE option in SOCKETS_SetSockOpt function. */
     uint32_t ulServerCertificateLength; /**< Length of the server certificate. */
+    #if ( configPLATFORM_SOCKET_UDP_SUPPORT == 1 )
+        uint16_t usLocalPort;
```

Comment:
It is great that it is documented in the PR that ST does not support bind, but can we write that in the comments too?

Reply:
I also think that will be a good idea as it will be easier to find the comments in the code while reading the code rather than finding the PR documentation